lib/runtime.js

```diff
@@ -65,6 +61,23 @@ export default class Runtime {
     const executionOptions = !options ? this.scriptOptions : options
     const commandContext = CommandContext.build(this, executionOptions, codeContext)
 
+    const terminal = this.terminals[0]
+    if (terminal) {
+      let command = commandContext.command
+      for (let i = 0; i < commandContext.args.length; i++) {
+        command += ` "${commandContext.args[i]}"`
+      }
+      terminal.run([`printf "\\33c\\e[3J" && ${command}`])
```

Comment:
Ya I'm not exactly sure what this command is trying to do I took it from the original changes in this PR. This obviously wouldn't work on windows since `printf` doesn't exist.

Reply:
I guess I'm not exactly sure what this PR is trying to accomplish. I have never used atom-script

Reply:
It seems to be related to cleaning the terminal.
https://stackoverflow.com/a/26615036/7910299

Reply:
>  I have never used atom-script

Check out some of the examples. There are some JavaScript files. Using `CTRL+B` to run code.

Reply:
We should probably just remove it. This should probably have tests written as well before merging.

```suggestion
      terminal.run([command])
```

Reply:
Hey I am pretty sure that was my code, and yes it was just to clear the terminal so that the output was cleaner. You can safely remove the printf part, or adapt it to work on windows. I don't use windows so didn't actually realise that printf wouldn't work there.

Reply:
I think as long as the output is written in a new line, we don't need to clean the terminal.